src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -431,6 +444,16 @@ public Index GetIndex(string name)
 
                 return instance;
             }
+            catch (TimeoutException toe)
```

Comment:
Should this be applied to CreateIndex(static) also?

Reply:
yes

Reply:
I'm not sure which overload you're referring to here, is it public async Task<Index> CreateIndex(IndexDefinition definition)?